ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -115,6 +116,26 @@ Value *getYkCtrlPointVarsStruct(Module *AOTMod, InputTrace &InpTrace) {
   return CI->getArgOperand(YK_CONTROL_POINT_ARG_IDX);
 }
 
+
+/// Extract all live variables that need to be passed into the control point.
+/// This is currently an overapproximation and will return some variables that
```

Comment:
shall we put `FIXME: ` before the second line?

Reply:
Fixed in dd3af2f